pkg/pmem-csi-driver/controllerserver-node.go

```diff
@@ -154,13 +218,15 @@ func (cs *nodeControllerServer) CreateVolume(ctx context.Context, req *csi.Creat
 		},
 	})
 
-	return &csi.CreateVolumeResponse{
+	resp = &csi.CreateVolumeResponse{
 		Volume: &csi.Volume{
 			VolumeId:           vol.ID,
 			CapacityBytes:      vol.Size,
 			AccessibleTopology: topology,
 		},
-	}, nil
+	}
+
+	return resp, nil
```

Comment:
nit (i.e. don't change it unless there is some other reason to touch the PR): this local `resp` variable seems unnecessary. The previous code without it should still work fine, shouldn't it?

Reply:
It helps in defer func, to determine if the call was a success or not, if not we revert the state.